fmriprep/workflows/util.py

```diff
@@ -18,14 +18,34 @@
 from niworkflows.nipype.interfaces import utility as niu
 from niworkflows.interfaces.utils import CopyXForm
 from niworkflows.nipype.interfaces import fsl, afni, c3, ants, freesurfer as fs
-from niworkflows.interfaces.registration import FLIRTRPT, BBRegisterRPT
+from niworkflows.interfaces.registration import FLIRTRPT, BBRegisterRPT, MRICoregRPT
 from niworkflows.interfaces.masks import SimpleShowMaskRPT
 
 from ..interfaces.images import extract_wm
 
 DEFAULT_MEMORY_MIN_GB = 0.01
 
 
+def compare_xforms(lta_list, norm_threshold=15):
+    import numpy as np
```

Comment:
Please add a docstring with a link to the original issue to get quickly to the discussion and examples of how the threshold plays out (e.g. 10 = very small affine changes accepted and 20 = large changes accepted). Something along these lines.

```Python
"""
Computes a *distance* between two transforms as the maximum overall displacement of 
the midpoints of the faces of a cube due to translation and rotation.

Parameters

  lta_list : list or tuple of str
      the two given affines in LTA format

  norm_threshold : int
      the upper bound limit to the distance of the second transform w.r.t. the first; the lower
      the smaller distances are accepted; a default of 15 was found empirically appropriate
      to decide whether BBR transforms should be rejected.

"""
```

Reply:
Added link to original issue.